src/redux/actions/ForgotPasswordAction.js

```diff
@@ -0,0 +1,41 @@
+import axios from "axios";
+import * as ActionTypes from "../action_types";
+
+/* request to change password
+*
+* @param {string} email
+* @return {object} type and responseData
+*/
+export const forgotPassword = responseData => ({
+  type: ActionTypes.FORGOT_PWD,
+  responseData
+});
+
+/* error from forgot password request
+*
+* @param {string} error
+* @return {object} type and responseData,{boolean} info
+*/
+
```

Comment:
Remove this extra line after the comment. And your asterisks are placed incorrectly, please ask me in person.